LEGO1/lego/legoomni/src/worlds/registrationbook.cpp

```diff
@@ -440,7 +454,15 @@ void RegistrationBook::ReadyWorld()
 		}
 	}
 
-	if (m_infocenterState->m_letters[0] != NULL) {
+#ifdef BETA10
+	InfocenterState* infocenterState = (InfocenterState*) GameState()->GetState("InfocenterState");
+	assert(infocenterState);
+
+	if (infocenterState->FirstLetterIsNotNull())
+#else
+	if (m_infocenterState->FirstLetterIsNotNull())
```

Comment:
Adding either the assertions above or changing this call causes the match to get worse. Adding both appears to be ok, though.